api/v1alpha1/oadp_types.go

```diff
@@ -406,8 +406,11 @@ func (dpa *DataProtectionApplication) AutoCorrect() {
 		if pvOperationTimeout, err := time.ParseDuration(resticTimeout); err == nil && dpa.Spec.Configuration.Velero.Args.PodVolumeOperationTimeout == nil {
 			dpa.Spec.Configuration.Velero.Args.PodVolumeOperationTimeout = &pvOperationTimeout
 		}
-	}
-	if dpa.Spec.Configuration.Velero.Args != nil && dpa.Spec.Configuration.Velero.Args.RestoreResourcePriorities == "" {
+		if dpa.Spec.Configuration.Velero.Args.RestoreResourcePriorities == "" {
```

Comment:
I think you may still need to handle the Args nil check so that we don't have a nil pointer deference.

If it is nil, we are going to have to set it though.

Reply:
@mateusoliveira43 talked to me offline